### ISSUE TITLE
[Editor] Empty paragraph returned instead of empty string / null

### ISSUE THREAD

**Issue by philippbuerger** [livewire/flux#816]
### Flux version

v1.0.29

### Livewire version

v3.5.16

### What is the problem?

At the moment, if you remove the content of an editor completely, it remains an empty `<p>` in the content so for e.g. a frontend check, the saved content is not fully empty. Of course one solution is, to call `strip_tags` on the string and check for yourself if it is empty. On the other hand tiptap provides a function for that:

https://tiptap.dev/docs/editor/api/editor#isempty

So in the implementation this check could be done before returning the content to Livewire and return an empty string / null instead of the empty `<p>`.

For testing just use my snippet, enter some content, remove it again and save. You should see the empty `<p>` tag in the dd

### Code snippets

```blade
<?php

use function Livewire\Volt\{rules, state, layout};

state([
    'mail_footer' => ''
]);


$update = function () {
    dd($this->mail_footer);
}

?>

<div>
    <form
        wire:submit.prevent="update"
        class="space-y-6"
    >
        <flux:editor
            wire:model="mail_footer"
            label="{{ __('Email footer') }}"
            description="{{ __('Will be shown on the bottom of every email.') }}"
            toolbar="bold italic underline strike | align | link ~ undo redo"
        />

        <flux:button
            type="submit"
            variant="primary"
        >
            {{ __('Save') }}
        </flux:button>
    </form>
</div>
```

### How do you expect it to work?

Empty editor content should be empty instead of an empty `<p>` tag

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by jeffchown** [comment]
@philippbuerger Nice find!  And great snippet, thanks. (Just needed to remove your `unset-all-marks`)

I also like your suggested solution re: `flux:editor` utilizing TipTap's `isEmpty` to check the editor's value and return an empty string if indicated @calebporzio @joshhanley 

That said, as a double-check, I usually sanitize, then check the returned value of rich text/HTML textarea controls just in case - I've run into issues with other UIs/js editors.

**Comment by philippbuerger** [comment]
@jeffchown thank you! I updated my snippet. Sorry for that

**Comment by joshhanley** [comment]
@philippbuerger thanks for the report! Yep I can confirm this is happening. I think it would make sense to use `isEmpty` by default to clear the returned string.

Here's a simpler reproduction

```blade
<?php

use Livewire\Volt\Component;

new class extends Component {
    public $content;
}

?>

<div>
    <flux:editor wire:model.live="content" />

    {{ $content }}
</div>
```

**Comment by joshhanley** [comment]
I've submitted a PR that adds a check if the editor isEmpty and returns `null` if so.

**Comment by joshhanley** [comment]
This has been released in v1.0.30. It now returns an empty string `""` instead of `null` (that I had said above), due to native inputs return empty strings.
